mlir/utils/jenkins/Jenkinsfile

```diff
@@ -848,20 +840,37 @@ pipeline {
                                 sh """python3 ./bin/perfRunner.py --op=gemm --batch_all \
 					--configs_file=${WORKSPACE}/mlir/utils/performance/bert-configs \
 					--tuning_db=${WORKSPACE}/mlir_tuning_${CHIP}.tsv"""
-                                sh """python3 ./bin/perfRunner.py --op=gemm --batch_all \
-                    --configs_file=${WORKSPACE}/mlir/utils/performance/bert-configs \
-                    --tuning_db=${WORKSPACE}/mlir_tuning_${CHIP}.tsv --external-gemm-library CK"""
                             }
                         }
                     }
-                    stage("Copy earlier performance results") {
+                    stage("Test MLIR vs CK") {
```

Comment:
I'd like a `when` clause with 

> equals expected: true, actual: params.checkCK;

This will help CK to be less disruptive in our CI: we can turn it in jenkins build parameters off as soon as it starts to block our CI.

Reply:
Yes, good idea!